graphene will fail pkgconf test pipeline

```2024/11/12 23:11:03 INFO running step "test/pkgconf"
2024/11/12 23:11:03 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:03 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:03 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:03 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:03 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + basename /home/build/melange-out/graphene-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + dev_pkg=graphene-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + apk info -L graphene-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep -q ^Name: usr/lib/pkgconfig/graphene-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + basename usr/lib/pkgconfig/graphene-1.0.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + lib_name=graphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + echo usr/lib/pkgconfig/graphene-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep -q '^usr/lib/pkgconfig/graphene-1.0.pc$\|^usr/share/pkgconfig/graphene-1.0.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + pkgconf --exists graphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep -q ^Version: usr/lib/pkgconfig/graphene-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + pkgconf --modversion graphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 INFO 1.10.8 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep -q ^Libs: usr/lib/pkgconfig/graphene-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + pkgconf --libs graphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 INFO -lgraphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + grep -q ^Cflags: usr/lib/pkgconfig/graphene-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN + pkgconf --cflags graphene-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN Package gobject-2.0 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN Perhaps you should add the directory containing `gobject-2.0.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 WARN Package 'gobject-2.0', required by 'graphene-1.0', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:03 INFO pod 3dcae6cc72c05608e9baa8a69e5e3c9ac77e0a8db41839d28521b11a29230f15 terminated
2024/11/12 23:11:03 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:03 INFO   workspace dir: /home/user/tmp/melange-workspace-3124122686
2024/11/12 23:11:03 INFO   guest dir: /home/user/tmp/melange-guest-2563474309
2024/11/12 23:11:03 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/graphene] Error 1```

--- a/graphene.yaml
+++ b/graphene.yaml
@@ -46,7 +46,7 @@ subpackages:
     description: graphene dev
     test:
       pipeline:
-        - uses: test/pkgconf 
+        - uses: test/pkgconf
 
 update:
   enabled: true

--- a/graphene.yaml
+++ b/graphene.yaml
@@ -2,7 +2,7 @@
 package:
   name: graphene
   version: 1.10.8
-  epoch: 2
+  epoch: 3
   description: A thin layer of graphic data types
   copyright:
     - license: MIT
@@ -44,6 +44,9 @@ subpackages:
       runtime:
         - graphene
     description: graphene dev
+    test:
+      pipeline:
+        - uses: test/pkgconf 
 
 update:
   enabled: true

